Frontend/ui-common/src/Components/SourceToDestination/View.purs

```diff
@@ -120,9 +124,56 @@ sourceLayout config =
               , visibility $ config.horizontalSeperatorConfig.visibility
               , padding $ config.horizontalSeperatorConfig.padding
               ][]
+          , horizontalScrollView
```

Comment:
what is the requirement of this view here?

Reply:
used mainly on driver side as we need to show map here with a horizontal pill as well
as this is a common component